component-classes/index.js

```diff
@@ -50,14 +50,14 @@ export const slider = {
 };
 
 export const box = {
-  box: 'group block relative break-words last-child:mb-0 p-16 rounded-8', // Relative here enables w-clickable
+  box: 'group block relative break-words last-child:mb-0 p-16 rounded-8 s-text', // Relative here enables w-clickable
```

Comment:
These classes are also used in Expandable. At first glance it seems this should not create any text color overriding issues, but it would be good to double-check.

Reply:
As `s-text` is standard text color, I personally think that we should just remove it all together here...? There is no need setting it when it only will make it harder for consumers overriding it for the box content. Wdyt?

Reply:
I have nothing against it as long as it’s tested that the text color is still applied the way you say 😄

Reply:
Text color is inherited... so... 🤷😄 But if someone sets text color to blue on the parent element... then of course the box content will be blue... but I would expect it to be so, so I can't really see that it would be a problem? 🤷 😄 
...I have been looking into the styling of Expandable now.. and... WOW, that is a REAL mess... 🙄   `box` is added both to the component classes for expandable AND used inside the expandable component.
This is the problem with never "daring" to change anything... things will get messy... we need to start refactoring things. It is just styling, won't be the end of the world if the text color is wrong somewhere... then we can just fix it.

Reply:
> Text color is inherited... so... 🤷😄 But if someone sets text color to blue on the parent element... then of course the box content will be blue... but I would expect it to be so, so I can't really see that it would be a problem? 🤷 😄

Then if someone did that already and we change the way the box text color is set, updating the component package would result in potentially getting a different text color (except Elements where styles are isolated). Although may not be a big deal, would it make sense to mention it in the PR title for React and Vue when this change is pulled in, so it's apparent in the changelog that text color in Box is now inherited?

Reply:
Since `color` is inherited this will be the case for web components as well... unless they have their own reset. 🤔 😄

Reply:
Updated the PR description with this...